### DISCUSSION TITLE
Update about: GitHub Foundations Certification

### DISCUSSION THREAD

**Discussion by hasanshahriar32** [community/community#143443] [GitHub Education] [answered]
### Hi everyone,

Hi, 
I have a query about the scholarship on GitHub Foundations Certification exam for education pack holders. From the link to avail offers, it is now currently redirecting to a GitHub discussion page which is saying the offer to be renewed from oct 31st. As the due date have already passed, can anyone with the authority let us know the updates.

Thank you.

**Comment by davevad93** [comment, marked as answer]
Hi @hasanshahriar32 ,

Yes, technically they should have fixed it by October 31 but they haven't done it yet. I suggest you to subscribe to the discussion #138834 so you can stay tuned, as soon as the free certification is available again they will announce it there.
